spring-pulsar-spring-boot-autoconfigure/src/main/java/org/springframework/pulsar/autoconfigure/WellKnownAuthParameters.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright 2022 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.pulsar.autoconfigure;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Utility class to map Pulsar auth parameters to well-known keys.
+ *
+ * @author Alexander Preuß
+ */
+public final class WellKnownAuthParameters {
+
+	private static final Map<String, String> lowerCaseToCamelCase = new HashMap<>();
+
+	static {
+		// Athenz
+		lowerCaseToCamelCase.put("tenantdomain", "tenantDomain");
```

Comment:
I wonder if these are available in the auth modules? I'm not sure if I would want to pull them in as references even if they were available but it may be nice to just have a constant for the auth param (in camel case) then we could just use `toLowerCase` as the possibility for human error or having it in here 2x seems likely.  

Another option would be an enum for each that knows how to get its kebab case and lower case version. 

This is fine for now though. Just a couple of thoughts.

Reply:
>I wonder if these are available in the auth modules?
Some of them are, unfortunately not all of them...

You're right about the `toLowerCase`, I created a followup ticket (#111)